.gitignore

```diff
@@ -46,6 +46,7 @@
 ### Environment variables
 .env
 .env.*
+!.env.example
```

Comment:
Our instructions say use for the force add for this instead:
```
git add path/to/file --force
```

Reply:
Arriving back to this a while later.

My view is still that the file called `.env.example` should be ignored by default and this change should be pushed up into [the template](https://github.com/dxw/rails-template/blob/develop/.env.example)?

Reply:
I have opinions, but let's not block this with them any further.